spec/README.md

```diff
@@ -145,22 +147,21 @@ This had the advantage that encodings can be reused and implementation of encode
 |--------------|--------------------------------|--------------------------------------------|
 | Date         | Int32                          | number of days since Unix epoch            |
 | Timestamp    | Int64                          | number of milliseconds since Unix epoch    |
-| RangeMap     | RangeMap (Map<vec2<Double>, T> | for storing linear referencing information |
+| RangeMap     | RangeMap (Map<vec2<Double>, T>)| for storing linear referencing information |
```

Comment:
perhaps this for conciseness?   Also, note that C++ types might be a bit confusing for some. I, for example, not sure how `List<UInt8>` is different from `vec` - is there a type guide here?
 
```suggestion
| RangeMap     | Map<vec2<Double>, T> | for storing linear referencing information |
```

Reply:
`List` implies arbitrary length, `vec2` is always 2 items.

`vec2` is GLSL and Rust terminology, C++ would use `array<double,2>` (or `pair` or `tuple`).

I suppose familiarity with all these things is assumed, but it could be confusing.  There's a glossary at the end which needs to be filled out, and perhaps some of these should be added as well.